syntax = "proto3";

package structuredb.v1;

message BeginRequest {
}

message BeginResponse {
  string tx = 1;
}

message CommitRequest {
  string tx = 1;
}

message CommitResponse {
}

service Transactions {
  rpc Begin(BeginRequest) returns (BeginResponse) {}
  rpc Commit(CommitRequest) returns (CommitResponse) {}
}

